example/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "webpack-rails-example",
   "version": "0.0.1",
   "license": "MIT",
+  "scripts": {
+    "build":  "webpack -d --config config/webpack.config.js",
+    "start": "webpack-dev-server -p --config config/webpack.config.js"
```

Comment:
I would like to rename `start` to `serve`. People's apps might have other things they want to do with `npm start` that aren't webpack.